desktop/app/console-helper.js

```diff
@@ -16,7 +16,7 @@ let fileWritable = null
 // If the file is too big, let's reset the log
 if (logFile) {
   fs.access(logFile, fs.W_OK, err => {
-    if (err && err.errno !== -2 /* file does not exist */) {
+    if (err && (err.errno !== -2 && err.errno !== -4058) /* file does not exist */) {
```

Comment:
If the error is that the file doesn't exist, we can fix that by creating the file. If the error is anything else, we fail